iceoryx_hoofs/test/moduletests/test_cxx_algorithm.cpp

```diff
@@ -150,156 +153,177 @@ TEST_F(algorithm_test, MergeTwoDisjunctNonEmptySortedContainersWithAGap)
     ::testing::Test::RecordProperty("TEST_ID", "15d3c063-8bc5-47eb-84a4-35f055a1d82c");
     constexpr int64_t OFFSET = 41;
     constexpr int64_t GAP = 13;
-    vector<int64_t, 10U> a, b;
+    vector<int64_t, 10U> first;
+    vector<int64_t, 10U> second;
 
     for (int64_t i = 0; i < 5; ++i)
     {
-        a.emplace_back(i + OFFSET);
+        first.emplace_back(i + OFFSET);
     }
 
     for (int64_t i = 5; i < 10; ++i)
     {
-        b.emplace_back(i + OFFSET + GAP);
+        second.emplace_back(i + OFFSET + GAP);
     }
 
-    auto mergedContainer = uniqueMergeSortedContainers(a, b);
-    auto mergedContainerSwitched = uniqueMergeSortedContainers(b, a);
+    auto mergedContainer = uniqueMergeSortedContainers(first, second);
+    auto mergedContainerSwitched = uniqueMergeSortedContainers(second, first);
 
     ASSERT_THAT(mergedContainer.size(), Eq(10U));
     for (int64_t i = 0; i < 5; ++i)
-        EXPECT_THAT(mergedContainer[i], Eq(i + OFFSET));
+    {
+        EXPECT_THAT(mergedContainer[static_cast<uint64_t>(i)], Eq(i + OFFSET));
```

Comment:
I would prefer to make `i` `uint64_t` (or `uint32_t`) and avoid the cast. If e.g. `OFFSET` may need to be unsigned as well (and probably can without issues).